test/kernel_test.rb

```diff
@@ -918,18 +924,30 @@ def to_d
       describe 'when a string without numbers is given' do
         let(:input) { 'home' }
 
-        it 'raises error' do
-          exception = -> { Lotus::Utils::Kernel.BigDecimal(input) }.must_raise(TypeError)
-          exception.message.must_equal "can't convert #{input.inspect} into BigDecimal"
+        unless Lotus::Utils.jruby?
+          it 'raises error' do
```

Comment:
This test is bypassed because jruby doesn't raise any error in this case, but it should because MRI does that.